dbConfig.mjs

```diff
@@ -1,17 +1,14 @@
 import pkg from 'pg';
 const { Pool } = pkg;
+import dotenv from 'dotenv';
 
-const connectToDb = async () => {
-        try{const pool = new Pool({
-            user: process.env.DB_USERNAME,
-            host: process.env.DB_HOST,
-            database: process.env.DB_DATABASE_NAME,
-            password: process.env.DB_PASSWORD,
-            port: process.env.DB_PORT
-    });
-    await pool.connect()}
-    catch(error){
-        console.log(error)
-    }
-}
-export default connectToDb
\ No newline at end of file
+dotenv.config()
+const pool = new Pool({
+    user: process.env.DB_USERNAME,
+    host: process.env.DB_HOST,
+    database: process.env.DB_DATABASE_NAME,
+    password: process.env.DB_PASSWORD,
+    port: process.env.DB_PORT,
+})
+
+export default pool
```

Comment:
Any reason for changing this and using `pool` instead of using the `connectToDb()` function?

Reply:
Yes. It allows multiple users to access the database at the same time and helps to manage and reuse connections more efficiently.

Reply:
It also binds the queries to the available connections.

Reply:
Also, it's advisable to your config stuff inside the entry point of your application, which is `index.js`. Consider moving `dotenv.config()` back to where it was

Reply:
> It also binds the queries to the available connections.

Good catch